alcs-api/src/application/application.service.ts

```diff
@@ -46,31 +46,20 @@ export class ApplicationService {
   async createOrUpdate(
     application: Partial<Application>,
   ): Promise<Application> {
-    let applicationEntity = await this.applicationRepository.findOne({
+    const existingApplication = await this.applicationRepository.findOne({
       where: { fileNumber: application.fileNumber },
     });
 
-    // TODO: replace with AutoMapper
```

Comment:
👍  nice cleanup

Reply:
I started to add AutoMapper but realized it was from Application Entity to Application Entity to Object.assign was all that was needed.